packages/cosmos_ui_components/lib/cosmos_app_theme.dart

```diff
@@ -17,10 +17,7 @@ class CosmosAppTheme {
   static const radiusM = 8.0;
   static const radiusS = 4.0;
 
-  static const fontSizeXS = 12.0;
-  static const fontSizeS = 16.0;
-  static const fontSizeM = 20.0;
-  static const fontSizeL = 24.0;
+  static const fontSizeS = 10.0;
```

Comment:
`S` should stay as 12, in the future, if we ever need `10`, we will use `XS` name for that

Reply:
✅